### ISSUE TITLE
Topspeed not reading with Jaws for windows version 2026.2605.43

### ISSUE THREAD

**Issue by Dennisl123** [diamondStar35/top_speed#120]
Hi, running the latest version of Jaws it doesn't read any of the menu options.  I'm willing to test.  This is using the most recent public stable build.  If I need to test with another build please let me know.

**Comment by diamondStar35** [comment]
Hi.
@Dennisl123 
I believe somebody has a solution for your problem, since JAWS requires weird key hooking in order to work properly even though the speech library that we use, Prism, already supports it. But JAWS needs weird setup and key hooking installation in order to work with audio games in general. I can't provide any more info on this topic other than it's not the game's fault, but hopefully somebody can help you.

**Comment by Dennisl123** [comment]
if you can provide the solution i would appreciate it. Dennis Long Sent from my iPhone 16EOn Jun 14, 2026, at 7:34 AM, Nolan Roberts ***@***.***> wrote:﻿diamondStar35 left a comment (diamondStar35/top_speed#120)
Hi.
@Dennisl123
I believe somebody has a solution for your problem, since JAWS requires weird key hooking in order to work properly even though the speech library that we use, Prism, already supports it. But JAWS needs weird setup and key hooking installation in order to work with audio games in general. I can't provide any more info on this topic other than it's not the game's fault, but hopefully somebody can help you.

**Comment by MarcroSoft** [comment]
We need a keyhook in the game itself.
Maybe we can get some inspiration from the nvgt keyhook, I don't know, but I can look at it.

**Comment by spacemangaming** [comment]
@Dennisl123 yeah. I agree.

**Comment by Dennisl123** [comment]
if you could do this i would appreciate it. Dennis Long Sent from my iPhone 16EOn Jun 14, 2026, at 7:41 AM, MarcroSoft ***@***.***> wrote:﻿MarcroSoft left a comment (diamondStar35/top_speed#120)
We need a keyhook in the game itself.
Maybe we can get some inspiration from the nvgt keyhook, I don't know, but I can look at it.

**Comment by diamondStar35** [comment]
@MarcroSoft I have posted something like this on the Audio Games forum hoping that somebody could provide more info on key-hooking and why is it required with JAWS.

**Comment by Dennisl123** [comment]
Hoping this is fixed soon.  I would love to enjoy the game.

**Comment by Dennisl123** [comment]
When arrowing through things with Jaws it doesn’t read.  Another issue 
 Keyboard  to say I want to control the car via keyboard example turn left left arrow.  If this works with Jaws what TTS were you using in testing?  I tried with Eloquence.  I tried using Jaws 2025 and Jaws 2026.

**Comment by diamondStar35** [comment]
@Dennisl123 Sir, please don't repeatedly report the same issue. I am aware that the game doesn't work with JAWS, and JAWS is the only one to be blamed for it's stupid keyhooking system. 
You need to create a .jkm file for JAWS and I still don't know how to create that yet, or rather, haven't tested this solution.

**Comment by Dennisl123** [comment]
Hi, just wondering if there is any update on this?
